drivers/hid/hid-multitouch.c

```diff
@@ -1673,6 +1691,59 @@ static void mt_expired_timeout(struct timer_list *t)
 	clear_bit(MT_IO_FLAGS_RUNNING, &td->mt_io_flags);
 }
 
+static void update_keyboard_backlight(struct hid_device *hdev, bool enabled)
+{
+	struct usb_device *udev = hid_to_usb_dev(hdev);
+
+	unsigned char cmd_disable_backlight[] =
+		{0x24, 0xFF, 0x00, 0xFF, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+	unsigned char cmd_enable_backlight[] =
+		{0x24, 0xFF, 0x00, 0xFF, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+
+	unsigned char *cmd_buf;
+	unsigned char *dmabuf;
```

Comment:
The name `dmabuf` might be a bit misleading here.

Reply:
Yea, changed it to simply `buffer`